docs/Use cases/Air_quality_simulators_use_case.md

```diff
@@ -0,0 +1,115 @@
+# Creating an air quality monitoring - Coiote DM and Azure IoT Hub integration use case
+
+The Coiote DM and Azure IoT Hub integration lets you create
+
+## Prerequisites
+
+- An active Azure subscription.
+- An active Coiote DM account.
+
+## Creating and configuring an Azure IoT hub and storage account
+
+First you need to add a new IoT hub and a storage account in Azure. Here's how to do it:
+
+### Creating an IoT hub
+
+1. In your Azure portal home view, go to **IoT Hub** and select **Add**.
+
+    - In the **Basics** tab:
+        - select your subscription and resource group,
+        - pick your region,
+        - provide a name for your IoT hub.
+    - In the **Management** tab:
+        - in **Pricing and scale tier** select,
+        - optionally, turn off **Defender for IoT**.
+    - In the **Review + create** tab, click **Create**.
+
+### Creating a storage account
+
+While your new IoT hub is deploying, you can add a new storage account:
+
+1. In the Azure portal, go to **Storage accounts** and select **Add**.
+
+   - In the **Basics** tab:
+       - select your subscription and resource group,
+       - provide a name for your storage account,
+       - pick your location.
+   - In the **Review + create** tab, click **Create**.
+
+## Configuring the Azure IoT Hub integration extension
+
+Once the deployments are complete, go to Coiote DM to set up the Azure IoT Hub extension.
+If you haven't done this yet, please follow the [instruction for the Azure IoT Hub integration configuration](../Configuring_Azure_IoT_Hub_integration_extension).
+
+## Adding and connecting LwM2M air quality meter simulators to Coiote DM and Azure IoT Hub
+1. Go to your Azure IoT Hub and add new devices:
+    - Under **Explorers**, select **IoT Devices** and click **+ New**.
+    - Provide the name for your first device: ``air-quality-meter-example-0``.
+    - Click **Save**.
+    - Repeat the procedure for the other 5 devices (increase the number included in the device name).
+2. Go to Coiote DM and sync the previously added devices:
+    - In **Device inventory**, select **Sync with IoT platform -> Azure IoT Hub**.
+    - In the pop-up, click **Sync devices**.
+    - Devices should then be visible in **Device inventory**   
+3. Go to your command interpreter and register the device simulators:
```

Comment:
Maybe the command line is a better name? Or shell?

Reply:
I'll go for command line